app/controllers/publications_controller.rb

```diff
@@ -25,11 +27,11 @@ def index
     capActive = params[:capActive]
     page = params.fetch(:page, 1).to_i
     per = [params.fetch(:per, default_per).to_i, default_per].min
-    last_changed = Time.zone.parse(params.fetch(:changedSince, '1000-01-01')).to_s
+    last_changed = Time.zone.parse(params.fetch(:changedSince)) if params[:changedSince]
 
     if capProfileId.blank?
-      description = "Records that have changed since #{last_changed}"
-      matching_records = Publication.select(:pub_hash).updated_after(last_changed).page(page).per(per)
+      matching_records = Publication.select(:pub_hash).page(page).per(per)
```

Comment:
The sort order of the result set doesn't appear to be specified. Is it assumed to be consistent as an API client pages through the results?

Did we lose the filtering by `last_changed`? rialto-airflow isn't using it, but could some day if we ever get to doing incremental updates.

Reply:
1. I don't believe sort order is guaranteed.  My understanding is that in order to do efficient paging, Rails is going to do the sort order by primary key.  This will ensure the paging is consistent.  But it is worth investigating further, as if it wasn't being done correctly could explain both the duplicates and missing pubs when going over many pages.

2. The Profiles team does use this "changedSince" parameter, as they do incremental nightly harvests.  It's functionality is preserved in the line directly below (basically only doing the where clause by date if the user actually supplied a date).

Reply:
ok, reading further in the thread where this was mentioned, seems like it's kaminari, and seems like you guys figured this out, so ignore me 🤐